index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Tic tac toe</title>
+    <link rel="stylesheet" href="/css/style.css">
```

Comment:
path is not correct. remove extra '/' from this line.

Reply:
Thank you for the review, we've made the necessary changes